### DISCUSSION TITLE
Custom room/presence name?

### DISCUSSION THREAD

**Discussion by ktpx** [mKeRix/room-assistant#385] [Q&A] [answered]
Is there any way to set/override  the room name (for presence)?  Seems it only picks up the hostname, which makes for terrible room names. ;)  Ive not found anything in the docs about this.

thx,

**Comment by mKeRix** [comment, marked as answer]
You can set the `instanceName` as part of the [global options](https://www.room-assistant.io/guide/configuration.html#core-settings), which will then be used for the presence sensors. :)

**Reply by ktpx** [reply]
Thank you, that worked.
